llvm_ir_dataset_utils/compile_time_analysis_tools/python_processing/fitting_and_plotting.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: ascii -*-
+import numpy as np
+import matplotlib.pyplot as plt
+from read_column import open_and_load
+# Usage:
+# from fitting_and_plotting import plot_functionality
+
+
+def plot_functionality(lang: str, show: bool = False) -> None:
```

Comment:
This should be named something more appropriate about what it actually does.

Something like `plot_instructions_against_text_segment_size` would be better because it gives the consumer an idea of what it actually does rather than the developer having to reference the docstring/implementation.